tinker-actions/src/fde_dmv/enable_dmv_emt.sh

```diff
@@ -59,6 +59,9 @@ rootfs_roothash_size=50
 #Global var which is updated
 single_hdd=-1
 check_all_disks=1
+
+# Get the user provided lvm disk size number
+lvm_disk_size=$1
```

Comment:
Hardcode to 0 if and env variable is not passed to the tinker-action. 
LVM_DISK_SIZE